test/cppsuite/configs/search_near_01_stress.txt

```diff
@@ -0,0 +1,22 @@
+# Configuration for search_near_01 stress test.
+# The test will generate key_count_per_collection number of keys for each prefix in aaa -> zzz.
+# This config will have a 30 minute duration, with 3 tables of an entry for each prefix.
+duration_seconds=1800,
+cache_size_mb=1000,
+timestamp_manager=
+(
+    enabled=false,
+),
+workload_generator=
+(
+    populate_config=
+    (
+        collection_count=3,
+        key_count_per_collection=100,
+        key_size=5,
```

Comment:
We could potentially increase the `key_size` to force the keys to be split across multiple pages?

Reply:
I increases key_count_per_collection to achieve that. I think that is more reliable to reproduce that situation. Of course we can increase both.

Reply:
Please see my comment [here](https://github.com/wiredtiger/wiredtiger/pull/7151/files#r743363359). I am afraid we are not using this parameter, hence we will need to use the `key_size` field.

Reply:
Sorry for my previous comments (I deleted them), I misread the code. I am happy to keep it to `100` if that's enough to get the keys to be on multiple pages. Could you verify it?

Reply:
Maybe we could do both? Having a big key size and a lot of keys might be quite beneficial.

Reply:
I'm able to reproduce WT-7198 so I believe it is working.